cli/tests/pcluster/createami/test_pcluster_createami.py

```diff
@@ -100,11 +107,11 @@ def test_validate_createami_args_architecture_compatibility(
     )
     if error_expected:
         with pytest.raises(SystemExit) as sysexit:
-            createami._validate_createami_args_architecture_compatibility(args)
+            createami._validate_createami_args_ami_compatibility(args)
         assert_that(sysexit.value.code).is_not_equal_to(0)
```

Comment:
Similar comment about asserting on error messages if possible

Reply:
Still relevant, please fix

Reply:
The error is written to logger, do you know how to check it? I would suggest to keep the current code to maintain a consistent style throughout the file.